src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -72,32 +75,46 @@
 
   public ShooterSubsystem() {
     // Applies the previously-declared values to the PIDF controller.
-    shooterPID.setP(kP, 0);
-    shooterPID.setI(kI, 0);
-    shooterPID.setD(kD, 0);
-    shooterPID.setIZone(kIz, 0);
-    shooterPID.setFF(kFF, 0);
-    shooterPID.setOutputRange(-outputRange, outputRange, 0);
+    shooterPIDLeft.setP(kP, 0);
+    shooterPIDRight.setP(kP, 0);
+    shooterPIDLeft.setI(kI, 0);
+    shooterPIDRight.setI(kI, 0);
+    shooterPIDLeft.setD(kD, 0);
+    shooterPIDRight.setD(kD, 0);
+    shooterPIDLeft.setIZone(kIz, 0);
+    shooterPIDRight.setIZone(kIz, 0);
+    shooterPIDLeft.setFF(kFF, 0);
+    shooterPIDRight.setFF(kFF, 0);
+    shooterPIDLeft.setOutputRange(-outputRange, outputRange, 0);
+    shooterPIDRight.setOutputRange(-outputRange, outputRange, 0);
     // Sets the shooter motor to coast so that subsequent shots don't have to rev up
     // from 0 speed.
-    shooterMAX.setIdleMode(IdleMode.kCoast);
-    shooterMAX.setSmartCurrentLimit(currentLimit);
+    shooterMAXLeft.setIdleMode(IdleMode.kCoast);
+    shooterMAXRight.setIdleMode(IdleMode.kCoast);
+    shooterMAXLeft.setSmartCurrentLimit(currentLimit);
```

Comment:
Because these two motors are directly connected together, they cannot be controlled independently or their PIDs will fight.  One controller must be selected as the primary and the other configured as a follower.

This is the same situation as with the Falcons on the drive base.